uk.co.electronstudio.ComicReaderUltimate.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id" : "uk.co.electronstudio.ComicReaderUltimate",
+  "branch" : "1.0",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "21.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk" ],
+  "modules" : [ {
+    "name" : "openjdk",
+    "buildsystem" : "simple",
+    "build-commands" : [ "/usr/lib/sdk/openjdk/install.sh" ]
+  }, {
+    "name" : "ComicReaderUltimate",
+    "buildsystem" : "simple",
+    "build-commands" : [
+      "install -D cru.jar /app/bin/cru.jar",
+      "install -D cru.sh /app/bin/cru.sh",
+      "install -D icon128.png /app/share/icons/hicolor/128x128/apps/uk.co.electronstudio.ComicReaderUltimate.png",
+      "install -D icon64.png /app/share/icons/hicolor/64x64/apps/uk.co.electronstudio.ComicReaderUltimate.png",
+      "install -D uk.co.electronstudio.ComicReaderUltimate.desktop /app/share/applications/uk.co.electronstudio.ComicReaderUltimate.desktop",
+      "install -D uk.co.electronstudio.ComicReaderUltimate.appdata.xml /app/share/appdata/uk.co.electronstudio.ComicReaderUltimate.appdata.xml"
+    ],
+    "sources" : [
+      {
+        "type": "file",
+        "dest-filename": "cru.jar",
+        "url": "https://github.com/electronstudio/ComicReaderUltimate/releases/download/v1.0/CRU-1.0.jar",
+        "sha512": "1a351eebb4c2dc79a769a34dfa51a642504e748f5446a675744a8b31cb1707feee1ddd650bb61fd4c46cb5cd872575621cfdbcaebdc9d0e9dd3ee703170b4c71"
+      },
+      {
+        "type": "script",
+        "dest-filename": "cru.sh",
+        "commands": [
+          "java -Xms1G -Xmx4G -XX:+UseZGC -XX:MaxGCPauseMillis=10 -jar /app/bin/cru.jar \"$@\""
+        ]
+      },
+      {
+        "type": "file",
+        "path": "uk.co.electronstudio.ComicReaderUltimate.desktop"
+      },
+      {
+        "type": "file",
+        "path": "icon128.png"
+      },
+      {
+        "type": "file",
+        "path": "icon64.png"
+      },
+      {
+        "type": "file",
+        "path": "uk.co.electronstudio.ComicReaderUltimate.appdata.xml"
+      }
+    ]
+  } ],
+  "finish-args" : [
+    "--env=PATH=/app/jre/bin:/usr/bin",
+    "--socket=x11",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--filesystem=host"
```

Comment:
that's very broad permissions for a comic book reader.

Reply:
Yeah, probably something like `xdg-pictures` and `xdg-download` would be better here?

Reply:
If I double click on a CBZ file in my file manager then I need that file to open in my CBZ viewer.  That file could be located anywhere on the filesystem.  It wouldn't make sense that I could double click on a file and open it from one directory but just get a silent failure when I attempt the same thing from another directory.

For comparison, I just picked another similar app at random, Gwenview, and found it has very similar permissions, probably for same reasons: https://github.com/flathub/org.kde.gwenview/blob/master/org.kde.gwenview.json

Reply:
however i have changed it to host:ro